config/default_rules.yaml

```diff
@@ -73,7 +73,7 @@
                - "_recommended_labels_for_your_project"
 
     FROM: 
-      paramSyntaxRegex: /^[\w./\-:]+(:[${}\w.]+)?(-[${}\w.]+)?$/
+      paramSyntaxRegex: /^[\w./\-:]+(:[${}\w.]+)?(-[${}\w.]+)?( as \w+)?$/
```

Comment:
🙋 The test on ` as ` should probably be case insensitive, e.g. add `i` after `$/`.